src/lib.rs

```diff
@@ -93,17 +98,72 @@ use vec1::Vec1;
 
 /// Result of an Sdk operation
 pub type Result<T> = std::result::Result<T, IronOxideErr>;
+type PolicyCache = DashMap<PolicyGrant, Vec<WithKey<UserOrGroup>>>;
+
+/// IronOxide SDK configuration
+pub mod config {
+
+    #[derive(Debug, Clone, PartialEq, Eq, Hash, Serialize, Deserialize)]
+    pub struct IronOxideConfig {
+        pub(crate) policy_caching: PolicyCachingConfig,
+    }
+
+    impl IronOxideConfig {
+        pub fn new(policy_caching: PolicyCachingConfig) -> IronOxideConfig {
+            IronOxideConfig { policy_caching }
+        }
+    }
+
+    /// Policy evaluation caching config. Lifetime of the cache is the lifetime of the `IronOxide` struct.
+    ///
+    /// Since policies are evaluated by the webservice, caching the result can greatly speed
+    /// up encrypting a document with a [PolicyGrant](../policy/struct.PolicyGrant.html). There is no expiration of the cache, so
+    /// if you want to clear it at runtime, call [IronOxide::clear_policy_cache()](../struct.IronOxide.html#method.clear_policy_cache).
+    #[derive(Debug, Clone, PartialEq, Eq, Hash, Serialize, Deserialize)]
+    pub struct PolicyCachingConfig {
```

Comment:
I guess we make this now because there will be more configs later? Other option is flattening it into the IronoxideConfig.

Reply:
Yes. That was my thought. There will be more configs for invalidation.